src/adapter/src/catalog/consistency.rs

```diff
@@ -58,11 +65,14 @@ impl CatalogState {
         if let Err(dependencies) = self.check_object_dependencies() {
             inconsistencies.object_dependencies = dependencies;
         }
+        if let Err(items) = self.check_items() {
+            inconsistencies.items = items;
```

Comment:
Uncovered

Reply:
Part of the consistency checker, pulled out of this PR